application/controllers/admin/DataEntry.php

```
@@ -1738,14 +1800,22 @@ public function insert()
         $hasResponsesCreatePermission = Permission::model()->hasSurveyPermission($surveyid, 'responses', 'create');
-        $rawQuestions = Question::model()->findAll("sid = :sid", [":sid" => $surveyid]);
+        $rawQuestions = Question::model()->findAll([
+            'condition' => 'sid = :sid',
+            'params' => [':sid' => $surveyid],
+            'order' => 'question_order ASC',
+        ]);
 
         $questions = [];
+        $subquestions = [];
 
         $totalTime = 0;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$totalTime'. (undefined)

(UnusedLocalVariable)